jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -207,6 +209,7 @@ static void clinit_wrapper_do_recursive_calls(
   const irep_idt &class_name,
   code_blockt &init_body,
   const bool nondet_static,
+  const bool use_json_clinit,
```

Comment:
I don't like `json` in the name as the source format of the data is irrelevant; it could as well be read from an XML file  or whatever.

Reply:
I renamed this to `replace_clinit`. Similarly for other variable names, see my reply to @smowton 's comment below.